src/pages/groupComparison/MRNAEnrichments.tsx

```diff
@@ -45,6 +45,7 @@ export default class MRNAEnrichments extends React.Component<
             this.props.store.mrnaEnrichmentAnalysisGroups,
             this.props.store.studies,
             this.props.store.sampleKeyToSample,
+            this.props.store.copyNumberEnrichmentProfiles,
```

Comment:
why?

Reply:
I think it should be `mRNAEnrichmentProfiles`

Reply:
What did I do.... lost my mind...

Reply:
good catch @adamabeshouse !